src/services/API/Queries/Plio.js

```diff
@@ -0,0 +1,15 @@
+export function uniqueUsersQuery(plioId) {
+  // query to fetch the number of unique users for the given plio ID
+  // this is the query in the CubeJS Data Schema which is automatically
+  // converted to SQL by CubeJS: https://cube.dev/docs/getting-started-cubejs-schema
```

Comment:
Do we really need this?
`this is the query in the CubeJS Data Schema which is automatically converted to SQL by CubeJS`
Can keep the link for reference purposes